README.md

```diff
@@ -73,6 +73,7 @@ There are many available implementations to choose from, here are some options:
     * [`structured-logger`](https://docs.rs/structured-logger/latest/structured_logger/)
 * Complex configurable frameworks:
     * [`log4rs`](https://docs.rs/log4rs/*/log4rs/)
+    * [`logforth`](https://crates.io/crates/logforth)
```

Comment:
Perhaps I should add the demo to crate's doc and use the link to docs.rs instead.

Reply:
Updated in https://github.com/tisonkun/logforth/pull/29.

```suggestion
    * [`logforth`](https://docs.rs/logforth/*/logforth/)
```